sos/plugins/openstack_containerlogs.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2019 Red Hat, Inc., Cédric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class OpenStackCinder(Plugin):
```

Comment:
I'm assuming `OpenStackCinder` is a copy-paste error and should be named something else.

But also, you need to subclass the distro plugin classes as well, E.G.:

```python
class OpenStackContainers(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
```